neon_data_models/enum.py

```diff
@@ -92,3 +92,34 @@ class Weekdays(IntEnum):
     FRI = 4
     SAT = 5
     SUN = 6
+
+
+class SubmindStatus(Enum):
```

Comment:
what is our supported version of python? I believe for the recent ones using `StrEnum` for such cases is beneficial